roles/cs.aws-autoscaling/tasks/create.yml

```diff
@@ -1,29 +1,31 @@
-- name: Create Launch Configuration
-  ec2_lc:
+- name: Create Launch Template Configuration
+  ec2_launch_template:
     state: present
-    name: "{{ autoscaling_launch_configuration_name }}"
+    name: "{{ autoscaling_launch_template_name }}"
     region: "{{ aws_region }}"
-    assign_public_ip: "{{ autoscaling_assign_public_ip }}"
-    instance_type: "{{ autoscaling_instance_type }}"
     image_id: "{{ autoscaling_instance_ami_id }}"
-    security_groups: "{{ autoscaling_instance_security_groups }}"
-    instance_profile_name: "{{ autoscaling_instance_iam_role }}"
+    instance_type: "{{ autoscaling_instance_type }}"
+    iam_instance_profile : "{{ autoscaling_instance_iam_role }}"
     key_name: "{{ aws_ec2_ssh_key_name }}"
-    instance_monitoring: False
-    user_data: "{{ aws_ec2_instance_user_data }}"
+    network_interfaces:
+      - groups: "{{ autoscaling_instance_security_groups }}"
+        associate_public_ip_address: "{{ autoscaling_assign_public_ip }}"
+        device_index: 0
+        delete_on_termination: yes
+    user_data: "{{ aws_ec2_instance_user_data | b64encode }}"
```

Comment:
`user_data` need to come from `autoscaling_instance_start_script` (as previously)